pkgs/by-name/ti/tinfoil-cli/package.nix

```
@@ -0,0 +1,38 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  nix-update-script,
+}:
+
+buildGoModule rec {
+  pname = "tinfoil-cli";
+  version = "0.0.8";
+
+  src = fetchFromGitHub {
+    owner = "tinfoilsh";
+    repo = "tinfoil-cli";
+    tag = "v${version}";
```

could use finalAttrs
```suggestion
buildGoModule (finalAttrs: {
  pname = "tinfoil-cli";
  version = "0.0.8";

  src = fetchFromGitHub {
    owner = "tinfoilsh";
    repo = "tinfoil-cli";
    tag = "v${finalAttrs.version}";
```